pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://www.appcues.com
 repository: https://github.com/appcues/appcues-flutter-plugin
 environment:
   sdk: ">=2.16.2 <4.0.0"
-  flutter: ">=2.5.0"
+  flutter: ">=3.13.0"
```

Comment:
I'm assuming this will impose a new limitation on our consumers, but I think we need to be up to date here to support those on the more recent versions, as this functionality is broken if our older code is used on a newer flutter version (not just deprecated, but broken). Given that Flutter seems to have made breaking behavior changes as part of this update, I don't really see how we can support prior versions at this point.